staging/cse/windows/azurecnifunc.ps1

```diff
@@ -167,6 +160,16 @@ function Set-AzureCNIConfig
         }
     }
 
+    # Set the SDNRemoteArpMacAddress RegKey for HNS when AzureCNI is enabled

```

Comment:
will this script be invoked only once for VM lifetime?

Reply:
yes, it's the one that writes the conflist on Windows